docs-ref-conceptual/install-azure-cli-macos.md

```diff
@@ -34,9 +34,7 @@ brew update && brew install azure-cli
 
 > [!IMPORTANT]
 >
-> The Azure CLI has a dependency on the Homebrew `python3` package, and will install it.
-> The Azure CLI is guaranteed to be compatible with the latest version of `python3`
-> published on Homebrew.
+> The Azure CLI has a dependency on the Homebrew `python@3.10` package, and will install it.
```

Comment:
How often is the python version increased for azure cli? I'm asking because if the version is regularly updated, it might be easier to add more generic phrasing around the versions and instruct the user how to find out the version if necessary? 

I see the instructions below as well and if we do need to be specific, we just need to make sure to keep up on it as we increase the python version.

Reply:
> How often is the python version increased for azure cli?

Almost once per year, according to Python's release schedule.

Reply:
Should be fine then. We'll just make sure we keep it up to date so we don't have issues coming in because of outdated instructions. 

Thanks!